bosco-override-setup.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+# Populate the bosco override dir from a Git repo
+# Expected Git repo layout:
+#     RESOURCE_NAME_1/
+#         bosco_override/
+#         ...
+#     RESOURCE_NAME_2/
+#         bosco_override/
+#         ...
+#     ...
+
+function errexit {
+    echo "$1"
```

Comment:
```suggestion
    echo "$1" >&2
```

Consider sending error messages to stderr.  Not sure whether it matters for this script.